apps/test/netsim/NetSimRouterNode.js

```diff
@@ -373,45 +373,39 @@ describe("NetSimRouterNode", function () {
 
   describe("getConnections", function () {
     it ("returns an empty array when no wires are present", function () {
-      var wires;
-      routerA.getConnections(function (err, foundWires) {
-        wires = foundWires;
-      });
-      assert(wires !== undefined, "Set wires");
-      assertOwnProperty(wires, 'length');
+      var wires = routerA.getConnections();
+      assert(Array.isArray(wires));
       assertEqual(wires.length, 0);
     });
 
     it ("returns wires that have a remote end attached to the router", function () {
-      NetSimWire.create(testShard, 0, routerA.entityID, function () {});
-
-      var wires;
-      routerA.getConnections(function (err, foundWires) {
-        wires = foundWires;
-      });
-      assertEqual(wires.length, 1);
+      NetSimWire.create(testShard, {
+            localNodeID: 0,
+            remoteNodeID: routerA.entityID
+          }, function () {});
+      assertEqual(1, routerA.getConnections().length);
     });
 
     it ("returns NetSimWire objects", function () {
-      NetSimWire.create(testShard, 0, routerA.entityID, function () {});
-
-      var wires;
-      routerA.getConnections(function (err, foundWires) {
-        wires = foundWires;
-      });
-      assert(wires[0] instanceof NetSimWire, "Got a NetSimWire back");
+      NetSimWire.create(testShard, {
+            localNodeID: 0,
+            remoteNodeID: routerA.entityID
+          }, function () {});
+      assert(routerA.getConnections()[0] instanceof NetSimWire, "Got a NetSimWire back");
     });
 
     it ("skips wires that aren't connected to the router", function () {
-      NetSimWire.create(testShard, 0, routerA.entityID, function () {});
-      NetSimWire.create(testShard, 0, routerA.entityID + 1, function () {});
+      NetSimWire.create(testShard, {
+            localNodeID: 0,
+            remoteNodeID: routerA.entityID
+          }, function () {});
+      NetSimWire.create(testShard, {
+            localNodeID: 0,
+            remoteNodeID: routerA.entityID
+          } + 1, function () {});
```

Comment:
Great catch!

I updated `NetSimWire.create()` with argument validation so that this test (with typo intact) fails, then fixed the typo.